content/en/blog/2022-10-04-prove-the-authenticity-of-the-oci-artifacts/index.md

```diff
@@ -0,0 +1,232 @@
+
+The talk is enough ! Let's jump right into the details of how we can actually use it.
+
+Today, we'll be deploying [Kyverno](https://kyverno.io) project by storing its manifests in OCI registry packaged as an OCI Artifacts, thanks to _Flux CLI_, then we are going to sign it with _cosign_ tool and configure Flux to verify the artifacts’ signatures before they are downloaded and reconciled.
+
+We should have three things to complete this demo;
+
+* _cosign_ CLI
+  * <https://docs.sigstore.dev/cosign/installation/>
+* A Kubernetes cluster
+  * <https://kind.sigs.k8s.io/#installation-and-usage>
+* _Flux_ CLI
+  * <https://fluxcd.io/flux/cmd/>
+
+Let's start by creating a simple Kubernetes cluster:
+
+```shell
+kind create cluster
+```
+
+Let's install Flux on it, and I'll not go into the details of the installation process, you might use the [installation page](https://fluxcd.io/flux/installation/) of Flux to get more details about it.
+
+```shell
+export GITHUB_USER=developer-guy
+export GITHUB_TOKEN=$GITHUB_TOKEN
+
+flux bootstrap github \
+  --owner=$GITHUB_USER \
+  --repository=flux-gotk \
+  --branch=main \
+  --path=./clusters/my-cluster \
+  --personal
+```
+
+> ⚠️ Note: Don’t forget to change the values with your own details!
+
+Make sure that Flux is up and running:
+
+```shell
+$ flux check
+► checking prerequisites
+✔ Kubernetes 1.25.2 >=1.20.6-0
+► checking controllers
+✔ helm-controller: deployment ready
+► ghcr.io/fluxcd/helm-controller:v0.25.0
+✔ kustomize-controller: deployment ready
+► ghcr.io/fluxcd/kustomize-controller:v0.29.0
+✔ notification-controller: deployment ready
+► ghcr.io/fluxcd/notification-controller:v0.27.0
+✔ source-controller: deployment ready
+► ghcr.io/fluxcd/source-controller:v0.30.0
+► checking crds
+✔ alerts.notification.toolkit.fluxcd.io/v1beta1
+✔ buckets.source.toolkit.fluxcd.io/v1beta2
+✔ gitrepositories.source.toolkit.fluxcd.io/v1beta2
+✔ helmcharts.source.toolkit.fluxcd.io/v1beta2
+✔ helmreleases.helm.toolkit.fluxcd.io/v2beta1
+✔ helmrepositories.source.toolkit.fluxcd.io/v1beta2
+✔ kustomizations.kustomize.toolkit.fluxcd.io/v1beta2
+✔ ocirepositories.source.toolkit.fluxcd.io/v1beta2
+✔ providers.notification.toolkit.fluxcd.io/v1beta1
+✔ receivers.notification.toolkit.fluxcd.io/v1beta1
+✔ all checks passed
+```
+
+To be able to push Kyverno manifets into OCI registry, we should clone the Kyverno project first becaut it stores its manifests in a a file called [./config/install.yaml](https://github.com/kyverno/kyverno/blob/main/config/install.yaml):
```

Comment:
```suggestion
To be able to push Kyverno manifets into OCI registry, we should clone the Kyverno project first because it stores its manifests in a a file called [./config/install.yaml](https://github.com/kyverno/kyverno/blob/main/config/install.yaml):
```